src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -261,44 +270,50 @@ Dictionary
                   <div>
                     <div id="other-search">
                       <div className="form-check">
-                        Quick search &nbsp;&nbsp;
+                          Quick search &nbsp;&nbsp;
                         <form className="form-inline search-bar">
                           <i className="fas fa-search" />
                           <input
                             {
-                            ...getInputProps()
-                            }
+                              ...getInputProps()
+                              }
                             className="form-control search"
                             id="search"
                             placeholder="search"
                             aria-label="Search"
                           />
+                          {isLoading
+                              && <div className="ml-auto text-right">
+                                <Loader smaller />
+                              </div>
+                            }
                         </form>
                       </div>
                     </div>
                     <div className="search-ul">
                       {isOpen ? (
                         <ul {...getMenuProps()} className="search-ul">
                           {
-                          sourceConcepts.filter(item => !inputValue.trim()
-                          || item.display_name.toLowerCase()
```

Comment:
Is this just a formatting change?

Reply:
Yes, It was because I added the div for the loader component

Reply:
Yes it was on the search input
```
                     {isLoading
                              && <div className="ml-auto text-right">
                                <Loader smaller />
                              </div>
                            }
```